s1reader/s1_etad.ipynb

```diff
@@ -0,0 +1,438 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #22.        r1 = min(data1.shape[0], data2.shape[0])
</pre><p>Question: I'm not familiar with the radar grid in ETAD product, but what I'm understanding here is that we merge the two ETAD data (same location, different time), trim them to the same size, and subtract one by the another.</p><p> </p><p>can we assume that there is no shift between <code>data1</code> and <code>data2</code> in range and azimuth direction?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/opera-adt/notebooks/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
<p>Great question. We can not assume that. </p><p> </p><p>However, as noted in <code>s1etad</code> documentation and in the comment I wrote above, the <code>merge_correction()</code> is a very simple function for plotting purpose only, thus, the possible shift here is not important.  </p>

Reply:
@yunjunz Thanks for the clarification. I think it should be okay it that routine is for plotting purpose only, and you have mentioned that in the comment.